sources/packages/backend/libs/integrations/src/esdc-integration/msfaa-integration/msfaa-integration.module.ts

```diff
@@ -14,8 +18,14 @@ import { MSFAANumberService, SshService } from "@sims/integrations/services";
     MSFAARequestProcessingService,
     MSFAAResponseProcessingService,
     MSFAANumberService,
+    MSFAANumberSharedService,
+    SystemUsersService,
     SequenceControlService,
   ],
-  exports: [MSFAARequestProcessingService, MSFAAResponseProcessingService],
+  exports: [
+    MSFAARequestProcessingService,
+    MSFAAResponseProcessingService,
+    MSFAANumberSharedService,
```

Comment:
I believe the integration modulo should avoid exporting the services like `MSFAANumberSharedService`.
For this scenario, I believe that we would import it on queue-consumers module directly.